frame/nfts/src/types.rs

```diff
@@ -47,6 +49,11 @@ pub(super) type ItemTipOf<T, I = ()> = ItemTip<
 	<T as SystemConfig>::AccountId,
 	BalanceOf<T, I>,
 >;
+pub(super) type CollectionConfigFor<T, I = ()> = CollectionConfig<
```

Comment:
You use `Of` as suffix for the names above.

Reply:
It looks like we use both suffixes for types, e.g. `type ItemDetailsFor<T, I>`, though for the storages we always use the `Of` suffix. It would be nice to decide what suffixes we want to use for types, structures and storage.
What do you think about that?